### PR TITLE
Add info help message on no arg

### PR DESCRIPTION
**Pull request by heinst**
Added info help message when no argument is input for `bower info`. All others throw errors already, info was the only one that had no output when no input arg was given. (as far as I could find)

### CODE REVIEW

**Comment by sheerun** [comment]
A test please :)

**Comment by heinst** [comment]
@sheerun Added a test

File: lib/commands/info.js [hunk 1]
```diff
@@ -62,6 +62,10 @@ info.readOptions = function (argv) {
     var pkg = options.argv.remain[1];
     var property = options.argv.remain[2];
 
+	if (!(pkg || property)) {
```
Comment (contolini) [inline lib/commands/info.js]: No need to check for `property`. We only care if `pkg` is defined.

File: test/commands/info.js [hunk 2]
```diff
@@ -49,4 +50,23 @@ describe('bower info', function () {
...
+        })
+        .then(function(commandResult) {
+            // No work should have been done, so no value should be returned.
+            expect(commandResult).to.be(undefined);
+        })
+        .catch(function() {
+            expect().fail('should list info');
+        })
+        .done(done);
+    });
```
Comment (contolini) [inline test/commands/info.js]: `helpers.run` returns a [promise](https://github.com/bower/bower/blob/12258324d3962def121764be06f0b3025df0958a/test/helpers.js#L222-L235) so there's no need to import Q and roll your own. You can simplify this test by just doing:

```
it('shows help for info when no arg given', function () {
    return helpers.run(info).then(function(loggerEndData) {
        // No work should have been done, so no value should be returned.
        expect(loggerEndData[0]).to.be(undefined);
    });
});
```

Buuuuuut if you scroll up in the file you'll see that there's already a test doing the [same thing](https://github.com/bower/bower/blob/12258324d3962def121764be06f0b3025df0958a/test/commands/info.js#L32-L36). What you really want to do is check if `cli.createReadOptionsError` is getting called, so try something like:

```
it('throws when no arg given', function () {
    expect(function() {
        info.readOptions([])
    }).to.throwError(/info syntax error/);
});
```

Comment (sheerun) [inline test/commands/info.js]: Yup :) Could you fix it?

**Comment by vladikoff** [comment]
Please reopen the pull request if still want to add this 👍

**Pull request closed without merging**